Makefile

```diff
@@ -17,18 +25,44 @@ FFMPEG_OPTS ?= --prefix=$(FFMPEG_BUILD) \
                --enable-network \
                --enable-parser=h264 \
                --enable-parser=hevc
-
 CGO_LDFLAGS := -L$(FFMPEG_BUILD)/lib
-ifeq ($(UNAME_S),Linux)
+export PKG_CONFIG_PATH=$(FFMPEG_BUILD)/lib/pkgconfig
+
+ifeq ($(TARGET_OS),android)
```

Comment:
Could you please document in the readme what compile targets we support? Something like:

|   | linux | android | Darwin |
| ------------- | ------------- |-|-|
| X86_64  | ✅ | ✅ |❌   |
| aarm64  | ✅ | ❌ |  ✅ |


Also please document which specific versions of android & linux we support building from / to.